src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -386,15 +392,37 @@ void setOptimalIndex(int optimalIndex)
 
         RawPacket[] pktsOut = bitstreamController.rtpTransform(pktIn);
 
-        if (!ArrayUtils.isNullOrEmpty(pktsOut)
-            && pktIn.getSSRCAsLong() != targetSSRC)
+        if (REMOVE_VP8_EXTENDED_CONTROL_BITS
+            || (!ArrayUtils.isNullOrEmpty(pktsOut)
+            && pktIn.getSSRCAsLong() != targetSSRC))
         {
             // Rewrite the SSRC of the output RTP stream.
             for (RawPacket pktOut : pktsOut)
             {
                 if (pktOut != null)
                 {
                     pktOut.setSSRC((int) targetSSRC);
+
+                    if (REMOVE_VP8_EXTENDED_CONTROL_BITS)
+                    {
+                        // remove extended control bits
+                        REDBlock block = bitrateController.getVideoChannel()
+                            .getStream().getPrimaryREDBlock(pktOut);
+
+                        int shiftLen = DePacketizer
+                            .VP8PayloadDescriptor.getSize(block) - 1;
```

Comment:
I assume this is something for Edge. But it looks like it would brake the VP8 RTP payload format: if the X bit is set the length of the PD must be at least 2. Should we clear the X bit after removing the rest of the PD?